cli/src/utils/command.ts

```diff
@@ -61,6 +61,13 @@ export default abstract class BaseCommand extends Command {
     return headers;
   }
 
+  // Drops a return value or an exception from p, logging errors as
+  // debug messages.  Silences tslint "Promises must be handled
+  // appropriately".  Good e.g. for wrapping analytics calls.
+  protected drop<T>(p: Promise<T>, warnFn: (e: string | Error) => any = this.debug.bind(this)) {
+    p.then(() => undefined).catch(warnFn);
```

Comment:
> You don't need `then`, just `catch`

OK.  Hoped this would be clearer, clearly it isn't.

> Maybe collect all promises and wait for completion with timeout before process exit?

No, because I don't understand why.  What I think: If we want timeouts we should add them separately for the entire program.  A download can time out, not just an analytics report.  Meanwhile, there's no point in delaying when we report errors.